core/finality-grandpa/src/communication.rs

```diff
@@ -17,6 +17,7 @@
 //! Incoming message streams that verify signatures, and outgoing message streams
 //! that sign or re-shape.
 
+use std::collections::HashMap;
```

Comment:
This is not used?

Reply:
it is used, a merge earlier caused the build to break

Reply:
It is used. This branch did not compile before this change and I guess master ir broken too.